tensorflow_datasets/testing/quickdraw_test.py

```diff
@@ -0,0 +1,95 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for Quickdraw Sketch RNN data."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from tensorflow_datasets import testing
+from tensorflow_datasets.sequence import quickdraw
+import numpy as np
+import os
+import tensorflow as tf
+
+from tensorflow_datasets.core.utils import py_utils
+from tensorflow_datasets.testing import fake_data_utils
+
+
+def _output_dir():
+    return os.path.join(
+        py_utils.tfds_dir(),
+        "testing",
+        "test_data",
+        "fake_examples",
+        "quickdraw_sketch_rnn",
+    )
+
+
+def _gen_stroke():
+    delta_x = np.random.random_integers(-100, 100)
+    delta_y = np.random.random_integers(-100, 100)
+    pen_off = np.random.choice([0, 1])
+    return np.array([delta_y, delta_x, pen_off], dtype=np.int16)
+
+
+def _gen_sketch(max_strokes: int = 10):
```

Comment:
You should use `max_strokes=10` for python2 compatibility.

Reply:
Ouch, yea, totally forgot about Py2. Oopsie. I will need to check for f-strings too. :man_facepalming:

Reply:
Solved by 32209b3b356e5180d2d1abb0a4bc1eff2273a698